src/schemes/fluid/weakly_compressible_sph/system.jl

```diff
@@ -327,3 +341,39 @@ end
 @inline function correction_matrix(system::WeaklyCompressibleSPHSystem, particle)
     extract_smatrix(system.cache.correction_matrix, system, particle)
 end
+
+function compute_surface_normal!(system, surface_tension, v, u, v_ode, u_ode, semi, t)
+    return system
+end
+
+function compute_surface_normal!(system, surface_tension::SurfaceTensionAkinci, v, u, v_ode,
+                                 u_ode, semi, t)
+    (; cache) = system
+
+    # Reset surface normal
+    cache.surface_normal .= zero(eltype(cache.surface_normal))
+
+    @trixi_timeit timer() "compute surface normal" foreach_system(semi) do neighbor_system
+        u_neighbor_system = wrap_u(u_ode, neighbor_system, semi)
+        v_neighbor_system = wrap_v(v_ode, neighbor_system, semi)
+        nhs = get_neighborhood_search(system, semi)
+
+        calc_normal_akinci!(system, surface_tension, u, v_neighbor_system, u_neighbor_system, nhs,
+                            neighbor_system)
+    end
+    return system
+end
+
+@inline function get_normal(particle, particle_system::FluidSystem,
```

Comment:
We try to avoid `get_` functions since #68.
```suggestion
@inline function surface_normal(particle, particle_system::FluidSystem,
```

Reply:
Yes this was of course #61